lib/collection/src/update_handler.rs

```diff
@@ -328,10 +335,16 @@ impl UpdateHandler {
                                 stopped,
                             ) {
                                 // Perform some actions when optimization if finished
-                                Ok(result) => {
+                                Ok(num_points_optimized) => {
+                                    // FIXME: Decide on ordering
+                                    // ToDo: Only increment if it's an indexing optimization
+                                    total_indexed_points
+                                        .fetch_add(num_points_optimized, Ordering::Relaxed);
```

Comment:
To be more specific: all optimization variants can index. So we're specifically interested in whether the segment has an HNSW index or not. IIRC we can distinguish this by `Indexed` segment type.